polkadot/xcm/src/v3/traits.rs

```diff
@@ -25,6 +25,11 @@ pub use sp_weights::Weight;
 
 use super::*;
 
+// A simple trait to get the weight of some object.
+pub trait GetWeight<W> {
```

Comment:
If this hasn't changed between versions, then what we can do is re-export it from the older version.

Reply:
Maybe not worth making v3 dependant on v2 code.

Reply:
But we already do so with a lot of other types -- see the re-exports in the `v3/mod.rs` file.